docs/guides/ReactComponents.md

```diff
@@ -400,6 +400,51 @@ module.exports = {
 }
 ```
 
+### Adding a css pre-processor for sass
+
+Install `node-sass-chokidar` and `npm-run-all`.
+
+```sh
+npm install node-sass-chokidar npm-run-all --save-dev
+```
+
+Update your `package.json` accordingly:
+
+```json
+"scripts": {
+    "start": "npm-run-all -p build-css start-js",
```

Comment:
do you mean `watch-css` instead of `build-css`?

Reply:
this was a really long time ago. But i guess i mean `build-css`, because i created this repository using `nwb` and it has `build-css` and not `watch-css`

https://github.com/aviskarkc10/react-d3-donut/blob/master/package.json

I could very well be wrong.